clipper_admin/clipper_admin/sample_config.yaml

```diff
@@ -0,0 +1,42 @@
+models:
+  Model1:
```

Comment:
This is an invalid model name:

```
clipper_admin.exceptions.ClipperException: Invalid value: Model3: a model name must be a valid DNS-1123  subdomain. It must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?\Z'
```

use `model1` (lowercase `m`) instead.

Reply:
Done